### PR TITLE
fix: hot fixes for get started design page

### LINKED ISSUE #1133: Get started designing page content fixes
- [ ] Remove doubled titles: Core design kits Local design kits
- [ ] Update images
- [ ] Remove extra Designing section in side-nav due to merge conflict

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #1133 

## TESTING:  

- http://localhost:3000/designing/get-started
- [x] Remove doubled titles: `Core design kits` `Local design kits`
- [x] Update images
- [x]  Remove extra `Designing` section in side-nav due to merge conflict
- [x]  Create object for sponsor names
- [x]  Add dynamic captions
- [x]  Tool switcher should extend at md and sm

### CODE REVIEW

**Review by francinelucca — APPROVED** [review]
LGTM!

**Review by alisonjoseph — APPROVED** [review]
LGTM!

File: services/web-app/pages/designing/get-started.mdx [hunk 1]
```diff
@@ -145,8 +145,6 @@ Everyone using Carbon should use core design kits as a starting point for buildi
 This brings consistency and speed, allowing teams to focus on solving more specific user tasks. Core
 design kits include elements, core components, wireframes, and guidelines.
 
-### Core design kits
-
 <FilterableDesignKitTable
   designKitsData={props.designKitsData}
   designTools={['Figma', 'Sketch', 'Adobe XD', 'Axure']}
```
Comment (aubrey-oneal) [inline services/web-app/pages/designing/get-started.mdx:150]: Can we add the caption text to each design tool? Figma can stay as is but we can link to the tutorials for the others. For example: 

`To learn more about installing Sketch Libraries visit the [Sketch tutorial](http://localhost:3000/designing/sketch).`

File: services/web-app/pages/designing/get-started.mdx [hunk 2]
```diff
@@ -145,8 +145,6 @@ Everyone using Carbon should use core design kits as a starting point for buildi
 This brings consistency and speed, allowing teams to focus on solving more specific user tasks. Core
 design kits include elements, core components, wireframes, and guidelines.
 
-### Core design kits
-
 <FilterableDesignKitTable
   designKitsData={props.designKitsData}
```
Comment (aubrey-oneal) [inline services/web-app/pages/designing/get-started.mdx:149]: All items in the table should either be links (to figma design kit previews) or downloads of design kits. This should all be defined in the schema.

File: services/web-app/pages/designing/get-started.mdx (hunk 1 above)
Comment (aubrey-oneal) [inline services/web-app/pages/designing/get-started.mdx:150]: The tool switcher should extend the full length of the table. Check in with Tracey (she was updating the redline)

File: services/web-app/pages/designing/get-started.mdx [hunk 3]
```diff
@@ -145,8 +145,6 @@ Everyone using Carbon should use core design kits as a starting point for buildi
 This brings consistency and speed, allowing teams to focus on solving more specific user tasks. Core
 design kits include elements, core components, wireframes, and guidelines.
 
-### Core design kits
-
 <FilterableDesignKitTable
```
Comment (aubrey-oneal) [inline services/web-app/pages/designing/get-started.mdx:148]: Add wireframe design kits to Sketch tab (they are in the schema so hopefully that isn't too hard to add)

[the remark this answers is not in the document]
Comment (aubrey-oneal) [reply services/web-app/pages/designing/get-started.mdx:148]: The elements design kits should also come first in the table (see redline, too).

[the remark this answers is not in the document]
Comment (aubrey-oneal) [reply services/web-app/pages/designing/get-started.mdx:148]: Figma tab is missing elements design kits

[the remark this answers is not in the document]
Comment (aubrey-oneal) [reply services/web-app/pages/designing/get-started.mdx:148]: The axure design kit 'Axure template' is labeled as UI in the redline

File: services/web-app/pages/designing/get-started.mdx [hunk 4]
```diff
@@ -180,8 +178,6 @@ design kits include elements, core components, wireframes, and guidelines.
 Additional design kits compliment and extend Carbon. For example, IBM.com provides components for an
 editorial approach. More kits can be found in the [design kit catalog]().
 
-### Local design kits
-
 <FilterableDesignKitTable
   designKitsData={props.designKitsData}
   designTools={['Figma']}
```
Comment (aubrey-oneal) [inline services/web-app/pages/designing/get-started.mdx:183]: Im step 4, delete duplicate 'Home base for the IBM Brand story, visual brand elements, guidelines, and assets.'

**Review by aubrey-oneal — DISMISSED** [review]
Hi Andrea, I found some changes to make. Happy to jump on a call to go through them (I wasn't able to leave inline code comments for everything).

File: services/web-app/pages/designing/get-started.mdx [hunk 5]
```diff
@@ -180,8 +178,6 @@ design kits include elements, core components, wireframes, and guidelines.
 Additional design kits compliment and extend Carbon. For example, IBM.com provides components for an
 editorial approach. More kits can be found in the [design kit catalog]().
 
-### Local design kits
-
 <FilterableDesignKitTable
```
[the remark this answers is not in the document]
Comment (andreancardona) [reply services/web-app/pages/designing/get-started.mdx:181]: will pull this in from teams

File: services/web-app/pages/designing/get-started.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (andreancardona) [reply services/web-app/pages/designing/get-started.mdx:148]: moving these comments to a new issue to update schema @aubrey-oneal

File: services/web-app/pages/designing/get-started.mdx (hunk 2 above)
Comment (andreancardona) [reply services/web-app/pages/designing/get-started.mdx:149]: @aubrey-oneal we move this to new issue

File: services/web-app/pages/designing/get-started.mdx (hunk 3 above)
Comment (andreancardona) [reply services/web-app/pages/designing/get-started.mdx:148]: @aubrey-oneal adding this to new issue

**Comment by aubrey-oneal** [comment]
- Axure widgets is missing in core design kits
- Design kits tagged as elements aren't showing up first but were you fixing that in another issue? I also see IBM Grid template in sketch showing up in a funky order.
- Other changes look good!

**Comment by andreancardona** [comment]
> * Axure widgets is missing in core design kits
> * Design kits tagged as elements aren't showing up first but were you fixing that in another issue? I also see IBM Grid template in sketch showing up in a funky order.
> * Other changes look good!

all of these will be fixed in a new issue!  :)

**Pull request merged**